### ISSUE TITLE
Scrollbar of a ChildWindow drawn into its host window, not the child window's area

### ISSUE THREAD

**Issue by LoicBaumann** [ocornut/imgui#6438]
**Version/Branch of Dear ImGui:**

Version: 1.89.2
Branch: Don't know

**Back-end/Renderer/Compiler/OS**
Using ImGuiNet / .net 7 / Windows

My assumption is if I create a Window, then inside I create a child window, if whenever the child window's content exceeds the available area, then a scrollbar is displayed to allow scrolling.

That's what is happening, but the scrollbar is drawn into the area of the host window, not inside the child window, and it doesn't seem right to me.

In the screenshot below the child window is the area that has a red border. The scrollbar of the child is drawn in the area I've highlight in green but I would expect it to be drawn in the area in orange.

Is this behavior the expected one? If so, is there a way for me to get what I'm looking for?
Basically what I want to achieve is having a window which contains a toolbar that is always visible, then a surface below (the child window) which content may exceed the area, then using a scrollbar to display the content we want.

**Screenshots/Video**
![bad](https://github.com/ocornut/imgui/assets/48723305/28e09bab-cea5-46a0-917b-91fe6ebc257e)

**Comment by ocornut** [comment]
It's happening because your child window is sized in a way:
- which exceed the size of the parent window, so parent has a scrollbar
- where all its contents fit in the child, so child doesn't have a scrollbar

I'm not sure what else to say. Without a repro I'm assuming it is a bug in your code or misunderstanding on your side.

**Comment by LoicBaumann** [comment]
Omg, now I get it...
The Red square was drawn with paint. But I don't know why, but I was on the assumption the child window's size should be the "virtual" one. 
Which is totally silly, and doing exactly what you said. 
Sorry for the waste of time on your side and thank you for the prompt reply !

**Comment by ocornut** [comment]
Well, glad there's no issue then :)
Note you can use Metrics/Debugger to confirm actual bounding box and scroll rang eof windows, if needed.

**Comment by LoicBaumann** [comment]
Oh thanks for the info, I saw this long time ago but totally forgot it existed.
